index.html

```diff
@@ -52,6 +54,164 @@ <h1> Hey There. <br> I'm Dennis <br>
+          <div class="bigButton">See Projects</div>
+        </div>
+        
+        <section id="seeProjects">
+          <div id="cardWork">
+            <div class="bg">
+              <div class="prjectDescription">
+                <span>Profesional Art <br/> Printing Data</span>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <div class="projectBtnGroup">
+                  <div class="projectBtn">html</div>
+                  <div class="projectBtn">bootstrap</div>
+                  <div class="projectBtn">Ruby</div>
+                </div>
+              </div>
+              <span class="actionBtn">See Project</span>
+            </div>
+          </div>
+
+          <div id="cardWork">
+            <div class="bg">
+              <div class="prjectDescription">
+                <span>Profesional Art <br/> Printing Data</span>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <div class="projectBtnGroup">
+                  <div class="projectBtn">html</div>
+                  <div class="projectBtn">bootstrap</div>
+                  <div class="projectBtn">Ruby</div>
+                </div>
+              </div>
+              <span class="actionBtn">See Project</span>
+            </div>
+          </div>
+
+          <div id="cardWork">
+            <div class="bg">
+              <div class="prjectDescription">
+                <span>Profesional Art <br/> Printing Data</span>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <div class="projectBtnGroup">
+                  <div class="projectBtn">html</div>
+                  <div class="projectBtn">bootstrap</div>
+                  <div class="projectBtn">Ruby</div>
+                </div>
+              </div>
+              <span class="actionBtn">See Project</span>
+            </div>
+          </div>
+
+          <div id="cardWork">
+            <div class="bg">
+              <div class="prjectDescription">
+                <span>Profesional Art <br/> Printing Data</span>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <div class="projectBtnGroup">
+                  <div class="projectBtn">html</div>
+                  <div class="projectBtn">bootstrap</div>
+                  <div class="projectBtn">Ruby</div>
+                </div>
+              </div>
+              <span class="actionBtn">See Project</span>
+            </div>
+          </div>
+
+          <div id="cardWork">
+            <div class="bg">
+              <div class="prjectDescription">
+                <span>Profesional Art <br/> Printing Data</span>
```

Comment:
- [ ] Kindly I recommend to you use  a header tag `hx` the project title ,This way will be semantically correct and the screen readers and other technologies will thank you. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.